build.sh

```diff
@@ -48,28 +63,29 @@ for repo in ${repos}; do
     if [ -d "${repo}" ]; then
         echo "${repo} is already cloned, updating"
         pushd "${repo}"
-        # Clean up the repo and make sure it's in sync with upstream master
-        git checkout master
+        # Clean up the repo and make sure it's in sync with upstream base branch
+        git checkout ${BASE_BRANCH}
         git clean -xdf
         git reset HEAD --hard
         git branch -D ${BRANCH}
```

Comment:
I may have missed it but I could not fine the ${BRANCH} definition

Reply:
Line 12 in the updated file.

Reply:
So I do not understand why the only place the ${BRANCH} is used is for deleting a branch ? who created it

Reply:
It's created here:
https://github.com/container-mgmt/cm-integration/blob/stable_builds/build.sh#L93

The branch has to be deleted and re-created because we want to remove any non-upstream commits before we apply the pending patches (so we don't try to apply the same PR twice, and so that it'd be possible to remove pending PRs from the list.